sw/device/lib/testing/test_framework/ujson_ottf.h

```diff
@@ -20,6 +21,36 @@ extern "C" {
  */
 ujson_t ujson_ottf_console(void);
 
+/**
+ * Deserialize a ujson message with a CRC.
+ * This macro will deserialize the message and then deserialize the CRC and
+ * check that it matches the CRC of the message. The CRC is itself deserialize
+ * as the ujson structure `ottf_crc_t`.
+ *
+ * @param deser_fn_ A pointer to the deserialization function generated by
+ * ujson_derive.
+ * @param uj_ctx_ A `ujson_t` representing the IO context.
+ * @param data_ A pointer to the data to send.
+ *
+ */
+#define UJSON_WITH_CRC(deser_fn_, uj_ctx_, data_)                    \
+  ({                                                                 \
+    ujson_crc32_reset(uj_ctx_);                                      \
+    LOG_INFO("waiting for usjon message...");                        \
```

Comment:
```suggestion
    LOG_INFO("waiting for ujson message...");                        \
```

Also, I wonder if these logs will be too noisy

Reply:
Good point, I will remove those print.